contracts/staking/NativeTokenStakingManager.sol

```diff
@@ -0,0 +1,47 @@
+// (c) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+// SPDX-License-Identifier: Ecosystem
```

Comment:
(nit) Confirm with a lawyer, but I think this should be `UNLICENSED` per [Solidity conventions for non-OSS licenses](https://docs.soliditylang.org/en/v0.8.26/layout-of-source-files.html#:~:text=If%20you%20do%20not%20want%20to%20specify%20a%20license%20or%20if%20the%20source%20code%20is%20not%20open%2Dsource%2C%20please%20use%20the%20special%20value%20UNLICENSED.) (_not_ `UNLICENSE` without the `D`). There are [specific license identifiers for SPDX](https://spdx.org/licenses/).